src/spectrecoff/Spectre/Status.fs

```diff
@@ -10,34 +10,26 @@ type CustomSpinner =
       Look: Look Option }
 
 type StatusOperation = StatusContext -> Task<unit>
+type StatusOperation<'Result> = StatusContext -> Task<'Result>
 
-let start statusText (operation: StatusOperation) =
-    task {
-        return! AnsiConsole
-            .Status()
-            .StartAsync(statusText, operation)
-    }
+type StatusOperationAsync = StatusContext -> Async<unit>
```

Comment:
Do you actually need this 'non-generic' version, since `unit` is just another type, so should be covered by hte generic one.

Reply:
Done